### ISSUE TITLE
Using AudioStreamer prevents AVAudioRecorder from working

### ISSUE THREAD

**Issue by mknippen** [mattgallagher/AudioStreamer#38]
Use case:

User is playing audio.
Audio is stopped.
Streamer is destroyed.
User goes to record, fails in the prepareToRecord method. No clear message is given.
This is only true on device, tested on 4.3 and latest iOS5 beta. 

I would be willing to supply code if needed.

**Comment by ajmsoftwares** [comment]
yes , confirmed

**Comment by t3rra** [comment]
I solved that issue by changing

UInt32 sessionCategory = kAudioSessionCategory_MediaPlayback;

to

UInt32 sessionCategory = kAudioSessionCategory_MediaPlayback | kAudioSessionCategory_PlayAndRecord;

in AudioStreamer.m (line # ~775). By the way, thank developer for this cool library! It definitely shortened my development time :) Thank you.

**Comment by mknippen** [comment]
yes, I ended up changing to just the  kAudioSessionCategory_PlayAndRecord and it worked great.

Thank You,
Matthew
